.github/workflows/reusable-autolog.yml

```diff
@@ -0,0 +1,112 @@
+name: reusable-autolog
+
+on:
+  workflow_call:
+  
+    secrets:
+      WORKFLOW_TOKEN:
+        required: true
+      USR_MEGA_1:
+        required: false
+      PWD_MEGA_1:
+        required: false
+      USR_MEGA_2:
+        required: false
+      PWD_MEGA_2:
+        required: false
+      USR_ONEDRIVE_1:
+        required: false
+      PWD_ONEDRIVE_1:
+        required: false
+      USR_ONEDRIVE_2:
+        required: false
+      PWD_ONEDRIVE_2:
+        required: false
+      USR_YAHOO_1:
+        required: false
+      PWD_YAHOO_1:
+        required: false
+        
+  
+    # inputs in this case are arguments passed from caller workflow
+    inputs:
+    
+      # name of folder to commit csv logfile to
+      folder:
+        required: true
+        type: string
+        
+      # account platform name to be reflected in commit message, e.g. OneDrive
+      platform-name:
+        default: ${{ inputs.folder }}
+        type: string
+        
+    
+  
+# Workflow level env variables:
+# Declare GitHub Personal Access Token 
+# and login credentials of each account here
+
+env:
+    WORKFLOW_TOKEN: ${{ secrets.WORKFLOW_TOKEN }}
+    USR_MEGA_1: ${{ secrets.USR_MEGA_1 }}
+    PWD_MEGA_1: ${{ secrets.PWD_MEGA_1 }}
+    USR_MEGA_2: ${{ secrets.USR_MEGA_2 }}
+    PWD_MEGA_2: ${{ secrets.PWD_MEGA_2 }}
+    USR_ONEDRIVE_1: ${{ secrets.USR_ONEDRIVE_1 }}
+    PWD_ONEDRIVE_1: ${{ secrets.PWD_ONEDRIVE_1 }}
+    USR_ONEDRIVE_2: ${{ secrets.USR_ONEDRIVE_2 }}
+    PWD_ONEDRIVE_2: ${{ secrets.PWD_ONEDRIVE_2 }}
+    USR_YAHOO_1: ${{ secrets.USR_YAHOO_1 }}
+    PWD_YAHOO_1: ${{ secrets.PWD_YAHOO_1 }}
```

Comment:
Passed the each variable from the `secrets` context into the workflow level `env`